docs/en/build/linux.md

```diff
@@ -84,6 +84,14 @@ conda install pytorch==1.8.0 torchvision==0.9.0 cudatoolkit=11.1 -c pytorch -c c
 export cu_version=cu111 # cuda 11.1
 export torch_version=torch1.8
 pip install mmcv-full==1.4.0 -f https://download.openmmlab.com/mmcv/dist/${cu_version}/${torch_version}/index.html
+</code></pre>
+    </td>
+  </tr>
+  <tr>
+    <td>onnx </td>
+    <td>Install onnx as follows. Refer to the <a href="https://github.com/onnx/onnx">guide</a> for details.
+<pre><code>
+conda install -c conda-forge onnx
```

Comment:
well, as conda always installs packages not wanted, `pip` is preferred I guess

Reply:
This line comes from https://github.com/onnx/onnx README.md ...

Reply:
ONNX is actually in [requirements](https://github.com/open-mmlab/mmdeploy/blob/master/requirements/runtime.txt)

Reply:
reverted.